azext_iot/common/utility.py

```diff
@@ -438,10 +438,14 @@ def is_iso8601_time(self, to_validate: str) -> bool:
             return False
 
 
-def ensure_min_version(cur_ver, min_ver):
-    from pkg_resources._vendor.packaging import version
+def ensure_min_version(min_ver):
```

Comment:
It would be nice either through docstrings and/or slightly altering the name that this is to ensure the min version of the iothub mgmt sdk.